config/locales/en/shared.yml

```diff
@@ -377,11 +377,9 @@ en:
         employment: Employment income
         employment_details: Details
         partner:
-          heading: Partner employment income
```

Comment:
This is worrying, it was only added in the prior ticket to allow Case Workers to get clarity on the differences between Client and Partner sections,  can we have a chat about the reasons for reversion?